web/client/src/library/components/progress/Progress.tsx

```diff
@@ -1,17 +1,25 @@
+import { isFalse } from '@utils/index'
 import clsx from 'clsx'
 
-export interface PropsProgress extends React.HTMLAttributes<HTMLElement> {
-  progress: number
-  delay?: number
-  duration?: number
-}
+const PROGRESS_START = 3
```

Comment:
What does this 3 mean?

Reply:
In some cases we want to start progress not from 0 to indicate that process started but no progress has been recorded yet. And `PROGRESS_START` sets starting point. 
![Screenshot 2023-11-28 at 10 07 05 AM](https://github.com/TobikoData/sqlmesh/assets/5644753/c9931f10-ce48-427f-9722-2f7f387f0513)